src/pages/iou/steps/IOUAmountPage.js

```diff
@@ -217,32 +215,14 @@ class IOUAmountPage extends React.Component {
                     styles.justifyContentCenter,
                 ]}
                 >
-                    <TouchableOpacity onPress={() => {
-                        if (this.props.hasMultipleParticipants) {
-                            return Navigation.navigate(ROUTES.getIouBillCurrencyRoute(this.props.reportID));
-                        }
-                        if (this.props.iouType === CONST.IOU.IOU_TYPE.SEND) {
-                            return Navigation.navigate(ROUTES.getIouSendCurrencyRoute(this.props.reportID));
-                        }
-                        return Navigation.navigate(ROUTES.getIouRequestCurrencyRoute(this.props.reportID));
-                    }}
-                    >
-                        <Text style={styles.iouAmountText}>
-                            {lodashGet(this.props.currencyList, [this.props.iou.selectedCurrencyCode, 'symbol'])}
-                        </Text>
-                    </TouchableOpacity>
-                    <TextInput
-                        disableKeyboard
-                        autoGrow
-                        hideFocusedState
-                        inputStyle={[styles.iouAmountTextInput, styles.p0, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
-                        textInputContainerStyles={[styles.borderNone, styles.noLeftBorderRadius, styles.noRightBorderRadius]}
-                        onChangeText={this.updateAmount}
-                        ref={el => this.textInput = el}
-                        value={formattedAmount}
+                    <TextInputWithCurrencySymbol
+                        formattedAmount={formattedAmount}
+                        onChangeAmount={this.updateAmount}
+                        onCurrencyButtonPress={this.navigateToCurrencySelectionPage}
                         placeholder={this.props.numberFormat(0)}
-                        keyboardType={CONST.KEYBOARD_TYPE.NUMBER_PAD}
-                        blurOnSubmit={false}
```

Comment:
Hello! 

While refactoring this code, `blurOnSubmit` must have been removed by mistake. This effectively reverted https://github.com/Expensify/App/pull/9064 which fixed #8362.

@mdneyazahmad can you please submit a follow-up PR
cc: @parasharrajat @mountiny

Reply:
Great catch, the PR is up here https://github.com/Expensify/App/pull/9351

Would any one of you @rushatgabhane or @parasharrajat be able to give this a quick test based on the PR Rushat linked to make sure the behaviour works fine and approve the PR, I will merge. Thanks!

Reply:
Merged, thanks!

Reply:
I missed it while fixing the merge conflict. Thanks @rushatgabhane for catching it early and @mountiny for creating the PR.

Reply:
No worries, this is easy to miss.